apps/front/src/app/(auth)/login/loginForm/loginForm.tsx

```diff
@@ -0,0 +1,78 @@
+'use client'
+import React from 'react'
+import FormInput from '@src/components/ui/formInput'
+import { CircularProgress, IconButton, InputAdornment } from '@mui/material'
+import { Visibility, VisibilityOff } from '@mui/icons-material'
+import { FormProvider } from 'react-hook-form'
+import useLoginForm from '@src/app/(auth)/login/loginForm/useLoginForm'
+import ClearIcon from '@mui/icons-material/Clear'
+
+const LoginForm = () => {
+    const {
+        showPassword,
+        handleClickShowPassword,
+        handleMouseDownPassword,
+        methods,
+        login,
+        loading,
+        error,
+        isCredentialsInvalid,
+    } = useLoginForm()
+
+    return (
+        <FormProvider {...methods}>
+            <form
+                className='flex flex-col items-center space-y-8 m-5 mt-10'
+                onSubmit={methods.handleSubmit((formv) => login(formv))}
+            >
+                <FormInput
+                    name='login'
+                    label='Login'
+                    color='primary'
+                    error={isCredentialsInvalid}
+                    required
+                />
+                <FormInput
```

Comment:
imo to mógłby być osobny komponent

Reply:
Chodzi o komponent typu PasswordInput, który zawiera w sobie FormInput ale też logikę do wyświetlania hasła